backend/tshark/capture_manager.py

```diff
@@ -0,0 +1,63 @@
+import subprocess
+import os
+from datetime import datetime
+
+class CaptureManager:
```

Comment:
Find a better name something like `PacketRecordManager` 

Record is better Capture refer to capture a criminal 

https://www.wordreference.com/enfr/capture